### ISSUE TITLE
four_gamete error 

### ISSUE THREAD

**Issue by jodyhey** [jaredgk/PPP#27]
got a vcf-related error this when running this command 

vcf_four_gamete.py --vcfs 2Pop_TV_phased/2Pop.TV_0.vcf.gz --fourgcompat --reti --right --numinf 3 --out 2Pop_TV_fgt/2Pop.TV_0.vcf

[W::vcf_parse] Contig 'chr10' is not defined in the header. (Quick workaround: index the file with tabix.)

**Comment by aewebb80** [comment]
This could be resolved in two ways:
1) `vcf_four_gamete.py` checks if a vcf.gz has an index. If it does not, it could create one with tabix, I believe I have a function somewhere.
2) We reheader the phased file. This might take longer as we would likely need to write something as `bcftools reheader` requires a text file. So that would require creating a temp header text document.

**Comment by aewebb80** [comment]
As for 1 above:
```
from pgpipe.bcftools import check_for_index, create_index
check_for_index(filename)
create_index(filename)
```

**Comment by aewebb80** [comment]
Resolved in 07b792970b6ebe939a80479669e5a2da3ffed20d.

We'll need to update the conda package.

**Comment by jodyhey** [comment]
I guess this is just a warning, as  it turns out the output vcf files are there
